<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>725</width>
    <height>413</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>COM-Port Scanner</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icons.qrc">
    <normaloff>:/icon/icons/connector-icon.gif</normaloff>:/icon/icons/connector-icon.gif</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTreeWidget" name="treeWidget">
      <column>
       <property name="text">
        <string>Port</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>725</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionClose_to_Tray"/>
    <addaction name="actionExit_2"/>
   </widget>
   <widget class="QMenu" name="menuScan">
    <property name="title">
     <string>Scan</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionScan"/>
    <addaction name="actionContinious_Scan"/>
    <addaction name="actionStop_Continious_Scan"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuScan"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionExit_2">
   <property name="text">
    <string>&amp;Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::WindowShortcut</enum>
   </property>
  </action>
  <action name="actionClose_to_Tray">
   <property name="text">
    <string>Close to Tray</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
  <action name="actionScan">
   <property name="text">
    <string>Single Scan</string>
   </property>
  </action>
  <action name="actionContinious_Scan">
   <property name="text">
    <string>Start Continuous Scan</string>
   </property>
  </action>
  <action name="actionStop_Continious_Scan">
   <property name="text">
    <string>Stop Continuous Scan</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
